fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -141,20 +143,30 @@ public class DatabaseTransactionMgr {
 
     // not realtime usedQuota value to make a fast check for database data quota
     private volatile long usedQuotaDataBytes = -1;
+    private long lockWriteStart;
+
+    private long lockReportingThresholdMs = Config.lock_reporting_threshold_ms;
 
     protected void readLock() {
+        long waitReadStart = System.currentTimeMillis();
         this.transactionLock.readLock().lock();
+        checkAndLogWaitTime(waitReadStart, System.currentTimeMillis());
```

Comment:
It's not necessary to log readlock wait time, long wait time is always caused by some thread holding writelock too long, and further more, if some thread hold's writelock for a long time, after it unlocked, there's lots of threads get the readlock, they all print such log, especially with stacktrace, which is quite slow.

Reply:
Thanks for comment, i have changed the code.
I agree to remove lock wait time, and i thinks it's only necessary to record the time of holding write lock, which is sufficient to reflect to situation of lock.